erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -55,7 +56,25 @@ def get_average_age(fifo_queue, to_date):
 
 	return flt(age_qty / total_qty, 2) if total_qty else 0.0
 
+def get_range_age(filters, fifo_queue, to_date):
```

Comment:
@AfshanKhan this logic is not enough. If have items in multiple age ranges but it just displays age only for the first item in FIFO queue. I have available qty as 3 and 2 but in ageing columns only 1 and 1 is shown i.e only fir the first item in the FIFO Queue

![image](https://user-images.githubusercontent.com/42651287/88178471-11689680-cc48-11ea-9995-4306ec154e57.png)

Reply:
fixed.